sdk/automanage/test-resources.bicep

```diff
@@ -0,0 +1,84 @@
+@minLength(12)
```

Comment:
I'm happy to see the bicep usage! Please just commit the bicep file and not the ARM template, as the latter is unnecessary and will result in two deployments (which will resolve idempotently to the same set of resources, but will take longer to run).

Reply:
Thanks for reviewing @benbp! I'm actually using the ARM template to create some Virtual machine within some of the test cases. Is it fine if I just rename the ARM template?

Reply:
Ah I see. Yes it should be fine as long as you rename it so our automation won't pick it up by convention.